taqueria-vscode-extension/src/lib/gui/SandboxesDataProvider.ts

```diff
@@ -286,66 +301,69 @@ export class SandboxesDataProvider extends TaqueriaDataProviderBase
 	private async getSandboxImplicitAccounts(
 		element: SandboxChildrenTreeItem,
 	): Promise<SandboxImplicitAccountTreeItem[]> {
-		const tzktBaseUrl = this.sandboxStates[element.parent.sandboxName]?.getTzKtBaseUrl(element.parent.sandboxName);
+		const tzktBaseUrl = await this.getTzKtBaseUrl(element.parent.sandboxName);
 		if (!tzktBaseUrl) {
 			return [];
 		}
-		const response = await fetch(`${tzktBaseUrl}/v1/accounts?type.ne=contract`);
-		tzktBaseUrl;
-		const data = await response.json();
-		const sandbox = this.observableConfig.currentConfig.config?.config.sandbox?.[element.parent.sandboxName];
-		const aliases = (sandbox === undefined || typeof sandbox === 'string' || !sandbox.accounts)
-			? []
-			: Object.keys(sandbox.accounts)?.map(key => ({ key, value: sandbox.accounts?.[key] }));
-		return (data as any[]).map(item => {
-			const alias = aliases.find(a =>
-				!!a.value && (typeof a.value === 'string' ? a.value === item.address : a.value.publicKeyHash === item.address)
+		try {
+			const sandboxAccounts = getSandboxAccounts(this.observableConfig.currentConfig, element.parent.sandboxName);
+			const tzktAccounts = await Promise.all(sandboxAccounts.map(async account => {
+				const data = await getAccountFromTzkt(tzktBaseUrl, account);
```

Comment:
If the number of accounts is non-trivial, say 20, this makes 20 calls to TzKt. Is there a way to batch these?

Reply:
I couldn't find a proper filter based on address for the base `getAccounts` API call 🧐

Reply:
![image](https://user-images.githubusercontent.com/5188263/210617326-7d07516b-ad8c-4f4e-a55c-2b6379a4fcfe.png)

Reply:
Yeah I found that alredy some time ago, thanks! 👍

Reply:
@AlirezaHaghshenas This does not work. This filter is for accounts (starting with `tz123..`) but we want to filter by contract address (starting with KT).